File: src/APIs/BibliotecaGamificada.Livros.Api/Models/LivroCadastroModel.cs
using BibliotecaGamificada.Livros.Comum.Enums;

namespace BibliotecaGamificada.Livros.Api.Models
{
    public class LivroCadastroModel
    {
        public string? id { get; set; }
        public string titulo { get; set; }
        public string autor { get; set; }
        public string? capa { get; set; }
        public Genero genero { get; set; }
        public string instituicao { get; set; }

        public LivroCadastroModel(string titulo, string autor, string capa, Genero genero, string instituicao)
        {
            this.titulo = titulo;
            this.autor = autor;
            this.capa = capa;
            this.genero = genero;
            this.instituicao = instituicao;
        }
    }
}

File: ClientApp/src/app/models/entidades/LivroCadastro.model.ts
import { Genero } from "../livro/Genero.model";

export class LivroCadastroModel{
    id?: string = "";
    titulo: string = "";
    autor: string = "";
    capa?: string = "../../../assets/images/default_capa.png";
    genero: Genero;
    instituicao: string = "";
    constructor(titulo: string,autor: string,genero: Genero,instituicao: string,capa?: string) {
        this.titulo = titulo;
        this.autor = autor;
        this.genero = genero;
        this.instituicao = instituicao;
        if(capa!=undefined)
            this.capa = capa;
    }
}